spec/models/user_spec.rb

```diff
@@ -1,3 +1,5 @@
+require "rails_helper.rb"
+
 RSpec.describe User, type: :model do
   it 'is valid with all required attributes' do
     user = User.new(
```

Comment:
We should keep a patter in our tests, and go with either `.new` or with `bot factories`. I suggest we go with factories since they are easier to change/mantain.

We can go with `build(:user)` here if we don't want to save it, and `create(:user)` when we do.

[Docs](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#build-strategies)

Reply:
Done ✅